src/energyplus/ForwardTranslator/ForwardTranslateCoilHeatingDXMultiSpeed.cpp

```diff
@@ -171,10 +171,15 @@ namespace energyplus {
       }
 
       // RatedSupplyAirFanPowerPerVolumeFlowRate2017
-      if ((value = stage.ratedSupplyAirFanPowerPerVolumeFlowRate())) {
+      if ((value = stage.ratedSupplyAirFanPowerPerVolumeFlowRate2017())) {
         eg.setDouble(Coil_Heating_DX_MultiSpeedExtensibleFields::SpeedRatedSupplyAirFanPowerPerVolumeFlowRate2017, value.get());
       }
 
+      // RatedSupplyAirFanPowerPerVolumeFlowRate2023
+      if ((value = stage.ratedSupplyAirFanPowerPerVolumeFlowRate2023())) {
+        eg.setDouble(Coil_Heating_DX_MultiSpeedExtensibleFields::SpeedRatedSupplyAirFanPowerPerVolumeFlowRate2023, value.get());
+      }
```

Comment:
the optional isn't needed (I know you're following the existing pattern).

```suggestion
      eg.setDouble(Coil_Heating_DX_MultiSpeedExtensibleFields::SpeedRatedSupplyAirFanPowerPerVolumeFlowRate2023, stage.ratedSupplyAirFanPowerPerVolumeFlowRate2023());
```